Server/image_functions/image_module/contrast_stretching.py

```diff
@@ -0,0 +1,53 @@
+def contrast_stretching(id1, id2):
+    """
+        Process the input image with contrast_stretching
+        and check the image size
+
+        :param id1: the input should be the name string of the image
+        :param id2: the input should be the name string of the processed image
+        :raises ImportError:  if input is not a string
+        :raises NameError:: if the id1 image file does not exist
+        :raises AssertionError: if the image array does not contain
+                three layers
+
+        :returns: a python dictionary contains the base64 string
+                  of the processed image and the image size
+        :rtype: python dictionary
+        """
+    try:
+        import PIL
+        from PIL import Image
+        import numpy as np
+        from skimage import exposure
+        import logging
+        from image_module.encode_image import encode_image
+    except ImportError:
+        print("Necessary imports failed")
```

Comment:
you need to return after this line

Reply:
Hey Anika, I have followed your suggestion to add the __pycache__ and .cache in my gitignore file. But whenever I do git add, those files will still be tracked by git.
<img width="442" alt="screen shot 2018-04-28 at 1 39 24 pm" src="https://user-images.githubusercontent.com/35426684/39399315-9e7a184c-4ae9-11e8-900d-0cd3e6aacefb.png">

Reply:
you need to add that full file path that I provided